azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/DeploymentType.java

```diff
@@ -63,11 +63,11 @@ public static DeploymentType fromString(final String input) {
     }
 
     // TODO: Change to lambda once on Java 8+
-    interface Function<T, R> {
+    interface Handler {
```

Comment:
The reason this interface was called Function<T, R> was because it would be simpler to refactor to Java 8 later. But It is no big deal anyways.

Reply:
I see.